Code/Framework/AzCore/AzCore/azcoretestcommon_files.cmake

```diff
@@ -7,6 +7,7 @@
 #
 
 set(FILES
+    UnitTest/DLLTestVirtualClass.h
```

Comment:
This should be in `Tests/azcoretestdll_files.cmake` file.

The `azcoretestcommon_files.cmake` file is used by the AzCoreTestCommon target which is not a dependency of AzCoreTestDLL.

Reply:
Updated, thanks!